tests/rptest/tests/scramful_eos_test.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2020 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+from ducktape.mark.resource import cluster
+
+from rptest.clients.types import TopicSpec
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.clients.rpk import RpkTool
+from time import sleep
+
+from confluent_kafka import (Producer, KafkaException)
+
+TOPIC_AUTHORIZATION_FAILED = 29
+TRANSACTIONAL_ID_AUTHORIZATION_FAILED = 53
```

Comment:
```
from kafka import errors as kerr
assert e.args[0].code() == kerr.TopicAuthorizationFailedError.errno
```

does that not work? then we don't have to duplicate what is already defined by the kafka protocol. the dependency on the kafka-python library should already be in the Dockerfile

Reply:
idk it feels wrong to mix two kafka libraries (kafka and confluent_kafka) in one file. Also I'm biased towards the latter because I've run into silent tx errors with the former so I tent to avoid using it.

Reply:
since kafka-python doesn't yet support transactions, I noticed that it doesn't define the named error constant for error `53`